sys/src/9/port/sysfile.c

```diff
@@ -856,7 +852,7 @@ write(int fd, void *p, int32_t n, int64_t off)
 
 	r = c->dev->write(c, p, n, off);
```

Comment:
Hmm. `r` is signed. What if it's -1 here?

Reply:
suicide?